docs/content/quickstart/installation.md

```diff
@@ -7,60 +7,60 @@ chapter = false
 
 ## Introduction
 
-The Kairos releases ships for user convenience a set of artifacts that can be used to install kairos on a node. 
-However, Kairos Kubernetes Native components allows the creation of these artifacts inside Kubernetes from a set of input images. 
+The Kairos releases ships for user convenience a set of artifacts that can be used to install Kairos on a node. 
+However, Kairos Kubernetes Native components allow the creation of these artifacts inside Kubernetes from a set of input images. 
 
-In the quickstart below we will use the artifacts generated by the kairos releases as we assume we don't have any prior Kubernetes cluster available.
+In the quickstart below, we will use the artifacts generated by the Kairos releases, as we assume we don't have any prior Kubernetes cluster(s) available.
 
 Most of the operations are completely automated, so there is little interaction needed with the Kubernetes components.
 
-Installation can be also completely driven by the Kubernetes Native components if you have already a Kubernetes cluster to manage installations, with zero touch configuration. See the [CRD quickstart (WIP)]().
+Installation can be also completely driven by the Kubernetes Native components if you already have a Kubernetes cluster to manage installations, with zero touch configuration. See the [CRD quickstart (WIP)]().
 
-The goal of this quickstart is to install kairos on a node and create a single-node Kubernetes cluster with [k3s](https://k3s.io).
+The goal of this quickstart is to install Kairos on a node and create a single-node Kubernetes cluster with [k3s](https://k3s.io).
 
-To simplify, we will install kairos inside a VM, however, your mileage and configuration may vary based on your setup, see the documentation for a more exhaustive list of examples.
+To simplify, we will install Kairos inside a virtual machine (VM); however, your mileage and configuration may vary based on your setup. See the documentation for a more exhaustive list of examples.
```

Comment:
What does mileage mean?

Reply:
'mileage may vary' means is used for saying that someone might have a different experience in a particular situation